resources/js/pages/game/[game]/requests.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTranslation } from 'react-i18next';
+
+import { SEO } from '@/common/components/SEO';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+import { GameSetRequestsRoot } from '@/features/games/components/+requests';
+
+const SetRequestorsPage: AppPage<App.Community.Data.GameClaimsCommentsPageProps> = ({ game }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <SEO
+        title={t('Set Requests - {{gameTitle}}', { gameTitle: game.title })}
+        description={t('A list of set requests for {{gameTitle}}.', { gameTitle: game.title })}
```

Comment:
nit: No need to translate this specific string. Descriptions are surfaced to search engines, where they'll always need to be in English.

Reply:
Will this reach search engines if it's behind a login page?